catroid/src/test/java/org/catrobat/catroid/test/formulaeditor/ParserTestObject.java

```diff
@@ -63,7 +63,9 @@ public void setUp() {
 		Project project = new Project(MockUtil.mockContextForProject(), "testProject");
 		ProjectManager.getInstance().setCurrentProject(project);
 		testSprite = new SingleSprite("sprite");
+		project.getDefaultScene().addSprite(testSprite);
 		ProjectManager.getInstance().setCurrentSprite(testSprite);
+		ProjectManager.getInstance().setCurrentlyEditedScene(project.getDefaultScene());
```

Comment:
Please remove this line entirely because of https://github.com/Catrobat/Catroid/blob/8e652d9b6883d530a2c1e1cfef408aa556e5b506/catroid/src/main/java/org/catrobat/catroid/ProjectManager.java#L463-L472

So the currently edited scene is set to the default scene already (auto*magic*ally :wink:).

```suggestion
```